### ISSUE TITLE
Improved search navigation suggestions

### ISSUE THREAD

**Issue by iconoclasthero** [johnfactotum/foliate#1323]
**Is your feature request related to a problem? Please describe.**
Use case: I listen to audiobooks and that's about it in terms of consuming books.  There are instances like today where I remember a quote from e.g., _Shantaram_ that I want to discuss at a meeting later.  I read the book a while ago and I'm trying to key in on a relatively common word, e.g., `feeling` and then look for the passage from there.  There is apparently no way to navigate the results  other than to repeatedly open and close the search window...which is a PITA.  Foliate lacks a straightforward forward/backward navigation common to mosl search features...either using the search menu or a keyboard shortcut.

**Describe the solution you'd like**
A keyboard shortcut to move back and forth on the search resultslist.

**Describe alternatives you've considered**
Not using Foliate to do this.  I could export to text and grep or use nano...  I mean vim does this too, but I don't use that.

**Additional context**
![Screenshot from 2024-05-26 10-15-01](https://github.com/johnfactotum/foliate/assets/535962/90911c07-c827-49f4-9bb4-eb4145a4ed28)

**Comment by iconoclasthero** [comment]
Oh, I just noticed that you can use `/` to get to the search window.
Potentially an easier fix is to do this...
`/` then have the cursor on the last selected search result instead of in the search box...  ot make that e.g., `^/` or `shift-/`...
anyway, instead of in the text box and then you can use the arrow keys to go to the next/previous hit.

hey, it's time to do something with 22.04, so looks like [I'll be able to compile Foliate once I do](https://github.com/johnfactotum/foliate/issues/1171).

**Comment by iconoclasthero** [comment]
So upon more investigation, `/` `↑` will get you to the list and then from there you can navigate with `↑`/`᪳᪳↓`.  If you use `/` `↓` you're stuck at the radio button.  I assume you don't have a choice on navigating down from the button, but if you could change that, that would be an improvement.

**tldr:  Found a workaround; an actual keyboard shortcut without having to go back to the search box would be preferred.**

**Comment by johnfactotum** [comment]
Duplicate of #693.

In v3, you can use <code><kbd>Ctrl</kbd><kbd>G</kbd></code> and <code><kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>G</kbd></code>.
